### PR TITLE
test: fixup the 1mops perf test

### PR DESCRIPTION
**Pull request by sergos**
The option --sync was ignored due to a misprint.
The temporary dir was never removed, overflowing the /tmp by default.

NO_TEST=test update
NO_DOC=test update
NO_CHANGELOG=test update

### CODE REVIEW

File: perf/lua/1mops_write.lua [hunk 1]
```diff
@@ -67,7 +80,7 @@ if params.help or params.h then
 end
 
 -- turn true to test the qsync
-local test_qsync = params.qsync or false
+local test_qsync = params.sync or false
```
Comment (ligurio) [inline perf/lua/1mops_write.lua:83]: Looks like a mess: in comment `qsync` and variable is `test_qsync`, but `params.sync`. Should we rename `qsync` to `sync` everywhere?

File: perf/lua/1mops_write.lua [hunk 2]
```diff
@@ -28,12 +28,25 @@ local params = require('internal.argparse').parse(arg, {
 
 local test_dir = fio.tempdir()
 
+local function rmtree(s)
```
Comment (ligurio) [inline perf/lua/1mops_write.lua:31]: Please add a check for root directory (`/`). It would be very-very bad if we will do `rm -rf /` on user's machine.

**Review by ligurio — COMMENTED** [review]
Thanks for the patch!

File: perf/lua/1mops_write.lua (hunk 1 above)
Comment (sergos) [reply perf/lua/1mops_write.lua]: donw

File: perf/lua/1mops_write.lua (hunk 2 above)
Comment (sergos) [reply perf/lua/1mops_write.lua:31]: It's a local function, never exposed to the user. The only use of the function is to remove the contents of the `tmp_dir` generated by the `fio.tempdir()`. Hence, no check is necessary.

File: perf/lua/1mops_write.lua (hunk 1 above)
Comment (ligurio) [reply perf/lua/1mops_write.lua]: Thanks!

File: perf/lua/1mops_write.lua (hunk 2 above)
Comment (ligurio) [reply perf/lua/1mops_write.lua:31]: Right, thanks!

**Review by ligurio — APPROVED** [review]
LGTM

**Review by igormunkin — APPROVED** [review]
@sergos, thanks for the patch! I'd rather implement the changeset via two separate patches, but I do not insist on it, hence LGTM.

File: perf/lua/1mops_write.lua [hunk 3]
```diff
@@ -28,12 +28,25 @@ local params = require('internal.argparse').parse(arg, {
...
+    end
+end
+
 local function exit(res, details)
     if (details ~= nil) then
         print(details)
     end
     if test_dir ~= nil then
-        fio.rmdir(test_dir)
+        rmtree(test_dir)
```
Comment (sergepetrenko) [inline perf/lua/1mops_write.lua:49]: There's `fio.rmtree()` and it seems to be doing the same thing: https://www.tarantool.io/en/doc/latest/reference/reference_lua/fio/#lua-function.fio.rmtree
Can you use it here?

Comment (sergepetrenko) [reply perf/lua/1mops_write.lua:49]: Ok, I see now. Thanks!

**Pull request merged**